bundles/scripting/sightly/js-use-provider/pom.xml

```diff
@@ -70,6 +70,7 @@
                     <instructions>
                         <Sling-Bundle-Resources>/libs/sling/sightly/js;path:=/SLING-INF/libs/sling/sightly/js</Sling-Bundle-Resources>
                         <Require-Capability>io.sightly; filter:="(&amp;(version&gt;=1.0)(!(version&gt;=2.0)))"</Require-Capability>
+                        <Import-Package>org.mozilla.javascript;version="[1.1,2)",*</Import-Package>
```

Comment:
Since SLING-6668 `o.a.s.scripting.javascript` is no longer embedding Rhino. @raducotescu and I think it's best we defined the version range manually rather than through the servicemix bundle (which BTW was resulting in `[1.7,2)` range, which is more restrictive than necessary)